java-runtime/src/main/proto/spine/validate/validation_error.proto

```diff
@@ -62,6 +62,28 @@ message ConstraintViolation {
     spine.validate.TemplateString message = 8;
 
     // The name of the validated message type.
+    //
+    // If the `(validate)` option is applied and a nested field contains an invalid value,
+    // this property holds the name of the root message, which triggered the validation.
+    //
+    // Example:
+    //
+    // ```
+    // message Student {
+    //     Contacts contacts = 1 [(validate) = true];
+    // }
+    // message Contacts {
```

Comment:
Please have empty lines between the examples. Yes, it would make the block bigger, but a code of an example should follow the normal style people write the code in the language of the example. We do have empty lines between type declarations in the production code.